cpp/include/cugraph_c/centrality_algorithms.h

```diff
@@ -235,23 +236,18 @@ cugraph_error_code_t cugraph_katz_centrality(const cugraph_resource_handle_t* ha
 /**
  * @brief     Compute betweenness centrality
  *
- * Betweenness can be computed exactly by specifying num_vertices as 0 and
- * vertex_list as NULL.  This will compute betweenness centrality by doing a
- * traversal from every vertex and counting the frequency that a vertex appears on
- * a shortest path.
+ * Betweenness can be computed exactly by specifying This will compute betweenness centrality by
```

Comment:
`specifying This`=>It seems like some words are missing between `specifying` and `This`.

Reply:
Cleaned up documentation